test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
Every change in this file is completely unrelated to the rest of the PR. Please revert these.